### DISCUSSION TITLE
ERR_ABORTED 520

### DISCUSSION THREAD

**Discussion by malogajski** [alpinejs/alpine#3511] [1. Help]
How to avoid this issue? It happens 1-2 per 60 days approx..

https://unpkg.com/alpinejs@3.x.x/dist/cdn.min.js net::ERR_ABORTED 520

**Comment by ekwoka** [comment]
Easy, just don't use unpkg.

unpkg is not safe for production use. It's for playing around. Not a CDN you should rely on. Put your scripts on your site.

**Reply by malogajski** [reply]
yes, that really makes sense. thanks for your reply!
